From bc7ee47c79700fcb6bd4fa9288f3082a116d1c2d Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Wed, 28 Oct 2015 22:14:42 +0100
Subject: [PATCH] WindowFeatures, Chrome: lower the minimum height to 5

* In case the height is expressed in GridUnits, it means it won't be possible to create a window with a height lower than 5gu.
* Chrome: lower minimum height to 5

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>
---
 chromium/third_party/blink/renderer/core/page/chrome_client.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chromium/third_party/blink/renderer/core/page/chrome_client.h b/chromium/third_party/blink/renderer/core/page/chrome_client.h
index f1a2e57b137..8241ff04391 100644
--- a/chromium/third_party/blink/renderer/core/page/chrome_client.h
+++ b/chromium/third_party/blink/renderer/core/page/chrome_client.h
@@ -411,7 +411,7 @@ class CORE_EXPORT ChromeClient
   virtual bool RequestPointerLock(LocalFrame*) { return false; }
   virtual void RequestPointerUnlock(LocalFrame*) {}
 
-  virtual IntSize MinimumWindowSize() const { return IntSize(100, 100); }
+  virtual IntSize MinimumWindowSize() const { return IntSize(100, 5); }
 
   virtual bool IsChromeClientImpl() const { return false; }
 
